### ISSUE TITLE
After module disabling, Content for ExternalCalendarEntry are not deleted

### ISSUE THREAD

**Issue by marc-farre** [humhub-contrib/external_calendar#71]
1. Enable the module
2. Add External Calendar Entries to a space
3. Disable the module globally
4. Go to the same space
5. The page crashes:

```
Could not load polymorphic relation! Class:
humhub\modules\external_calendar\models\ExternalCalendarEntry (Class
"humhub\modules\external_calendar\models\ExternalCalendarEntry" does
not exist)
````

**Comment by marc-farre** [comment]
@luke- https://github.com/humhub-contrib/external_calendar/pull/72

I didn't test this PR, but I had the issue on an instance after uninstalling the module.
If you think I should test it before merging into master, tell me.

On https://github.com/humhub-contrib/external_calendar/pull/72/files#diff-074f64cd19bc13bc527290951a58a65cb54f2f35a43ce5bf95e5dbb20b4d297dL61-L63, the lines where commented, I suppose because of timeout issues.

I've also replaced the `->all()` with `->each()` which could solve memory issues.

But now that module disabling is done by cronjobs, I think it's OK to put it back.

The only problem is with module uninstalling because of https://github.com/humhub/humhub/issues/6707
I'll try to work on it soon.

I saw that in `Module::disableContentContainer()` are not delete before deleting calendars, maybe we should add this deletion too? But here it won't be executed by cron jobs...
